pymobiledevice3/services/dvt/instruments/application_listing.py

```diff
@@ -1,10 +1,16 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from pymobiledevice3.services.dvt.dvt_secure_socket_proxy import DvtSecureSocketProxyService
 from pymobiledevice3.services.remote_server import MessageAux
 
 
 class ApplicationListing:
     IDENTIFIER = 'com.apple.instruments.server.services.device.applictionListing'
 
-    def __init__(self, dvt):
+    def __init__(self, dvt: 'DvtSecureSocketProxyService') -> None:
```

Comment:
Use `DvtSecureSocketProxyService`, not `'DvtSecureSocketProxyService'`

Reply:
k